src/app/lib/web_channel.js

```diff
@@ -0,0 +1,39 @@
+import Events from 'ampersand-events';
+
+class WebChannel {
+  constructor() {
+    // turn this into an event emitter
+    Events.createEmitter(this);
+
+    // listen for WebChannel messages
+    window.addEventListener('WebChannelMessageToContent', this._messageReceived.bind(this));
```

Comment:
@6a68 we need to sync up on what these should be called. Should we have something unique to idea town? 
`idea-town-web-channel-msg-2-content`